src/index.css

```diff
@@ -226,29 +245,44 @@ footer {
   background: none;
 }
 
+.comment-form {
+  width: 70%;
+  margin: 0 auto;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 8px;
+  background: none;
+  border-radius: 8px;
+}
+
 .username,
 .startdate,
 .enddate {
   width: 50%;
   padding: 5px;
-  border-radius: 1%;
+  border-radius: 8px;
   color: #000;
-  font-family: 'Roboto', sans-serif;
-  font-size: 10px;
+  font-family: "Roboto", sans-serif;
   border: none;
   box-shadow: 1px 1px 3px #000;
```

Comment:
It appears as if the styling properties for the reservation section were modified. If not, I believe while trying to fix your styling for the comment popup section, you unintentionally modified some styling for the reservation popup section and increased the font size for the reservation form. It has caused the page to appear as follows:







![kenneth2](https://github.com/ometman/JS-capstone-kanban/assets/129378066/f20540eb-060e-42cb-bdcd-5aa397c8fb72)


The reservation button has been shifted to the right. I don't think we need complex styling at this stage but let's try to keep the view very close to the wireframe as prescribed in the project requirement.